deploy/chart/values.yaml

```diff
@@ -19,6 +19,9 @@ olm:
   nodeSelector:
     beta.kubernetes.io/os: linux
   resources:
+    limits:
+     cpu: 400m
+     memory: 400Mi
```

Comment:
I see from the rest of your PR that you did not apply these limits to the upstream distribution. I believe that the next `make release` will apply them to upstream.

Reply:
[Here](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/deploy/chart/templates/0000_90_olm_01-prometheus-rule.yaml#L1) is an example where we don't add annotations to the olm namespace in upstream: